vars/main.yml

```diff
@@ -0,0 +1,5 @@
+# SPDX-License-Identifier: MIT
+---
+# Put the role's internal variables here that are not distribution specific.
+
+__certificate_provider_default: "certmonger"
```

Comment:
you don't need double quotes here

Reply:
efa00dad5ca6553db70463757df56dee022a20f6